### DISCUSSION TITLE
Revalidate Path externally

### DISCUSSION THREAD

**Discussion by LuizHAP** [vercel/next.js#65474] [App Router]
Hi!

Today we have a inbound CMS that control promotions (publishing, unpublishing, draft). Is it possibile with [revalidatePath](https://nextjs.org/docs/app/api-reference/functions/revalidatePath) to only revalidate a path (ex home) by a outside trigger?

Example: I have a cron on CMS, that check if a promotion is valid (by date), and today is the "live day" for it. My cron on CMS runs, I want to put in my home that published data. Can I create a path inside of Nextjs that runs `revalidatePath('/')`? Will work as inteded?

**Comment by victorcesae** [comment]
I'm facing a similar issue. Based on the documentation, if you put the root path will only revalidate the cache of the next page that your user access,
In case you are using the method in the backend i think nothing will be revalidated, because any page will be accessed next.
![image](https://github.com/vercel/next.js/assets/71783803/d061442f-a3fc-422a-8362-1ea4bb9c9d53)


My issue is that when my backend update some information, i will have to revalidate the tag of the query, but won't work if any page is accessed and not will revalidate the data for all users.
